.travis.yml

```diff
@@ -13,6 +13,10 @@ jobs:
     - node_js: lts/*
     - node_js: node
 
+    - stage: security scan
```

Comment:
security scan 🔐

Reply:
Where do you guys find these cool symbols?

Reply:
@davisjam If you type `:` you can start typing words and find emoji 🔍 

Otherwise, checkout https://emojipedia.org 😄